src/components/ModelingSidebar/ModelingPanes/MemoryPane.tsx

```diff
@@ -89,7 +89,7 @@ export const processMemory = (programMemory: ProgramMemory) => {
   const processedMemory: any = {}
   for (const [key, val] of programMemory?.visibleEntries()) {
     if (
-      (val.type === 'UserVal' && val.value.type === 'Sketch') ||
+      val.type === 'Sketch' ||
       // @ts-ignore
       (val.type !== 'Function' && val.type !== 'UserVal')
```

Comment:
All these conditions can be cleaned up, but the main thing is that there shouldn't be mention of `UserVal` anymore.

```suggestion
      val.type !== 'Function'
```